__tests__/pages/blog/Blog.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render} from "@testing-library/react";
+import Blog, { BlogPageProps } from "@/pages/blog";
+import { BLOG_TEXT, BLOG_TITLE } from "@/constants/constants";
+
+const props: BlogPageProps = {
+  categories: [
```

Comment:
Better to avoid static test data, try to use some library like faker to generate test data

Reply:
Please take a look at generating data and it’s customisation

Reply:
addressed, used faker to generate mock data for test cases